### PR TITLE
refactor config

### PR DESCRIPTION
**Pull request by beaugogh**

### CODE REVIEW

File: src/app/config/app.config.ts [hunk 1]
```diff
@@ -14,7 +14,24 @@ import {ErrorHelper} from '../utilities/error-helper';
 @Injectable()
 export class AppConfig {
 
+  public static DEFAULT_API_URL = 'https://transmart-dev.thehyve.net';
+  public static DEFAULT_API_VERSION = 'v2';
+  public static DEFAULT_APP_URL = 'http://localhost:4200';
+  public static DEFAULT_APP_VERSION = '0.0.1-SNAPSHOT';
```
Comment (gijskant) [inline src/app/config/app.config.ts]: Maybe the default app version should be something like 'no version specified'

File: src/app/config/app.config.ts [hunk 2]
```diff
@@ -14,7 +14,24 @@ import {ErrorHelper} from '../utilities/error-helper';
 @Injectable()
 export class AppConfig {
 
+  public static DEFAULT_API_URL = 'https://transmart-dev.thehyve.net';
```
Comment (gijskant) [inline src/app/config/app.config.ts]: Should this have a default? I would see this as always environment specific.

File: src/app/config/app.config.ts (hunk 1 above)
Comment (beaugogh) [reply src/app/config/app.config.ts]: good idea, will change

File: src/app/config/app.config.ts (hunk 2 above)
Comment (beaugogh) [reply src/app/config/app.config.ts]: yes, we should always specify this in config.default.json, on an off-chance that this url is not there, would be nice to have sth. to fall back on?

**Pull request merged**